core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-answer-group-modal.template.html

```diff
@@ -40,11 +40,12 @@ <h3>Add Response</h3>
     </div>
     <br>
 
-    <outcome-destination-editor outcome="tmpOutcome"
-                                ng-if="!questionModeEnabled"
-                                outcome-has-feedback="!isLinearWithNoFeedback(tmpOutcome)"
-                                add-state="addState">
-    </outcome-destination-editor>
+    <oppia-outcome-destination-editor [outcome]="tmpOutcome"
+                                      (get-changes)="getChanges()"
+                                      ng-if="!questionModeEnabled"
```

Comment:
Question: Is `ng-if` used here (instead of `*ngIf`) because the modal controller hasn't been migrated?

Reply:
Yes... thats correct.